src/vaadin-login-mixin.html

```diff
@@ -182,5 +182,17 @@
         }
       }
     }
+
+    _handleEvent(e) {
```

Comment:
would recommend something like `_retargetEvent` so the purpose of the method would be more clear